cmd/bbolt/main_test.go

```diff
@@ -93,6 +93,20 @@ func TestCheckCommand_Run(t *testing.T) {
 	}
 }
 
+func TestCheckCommandWithPageID_Run(t *testing.T) {
+	db := btesting.MustCreateDB(t)
+	db.Close()
+
+	defer requireDBNoChange(t, dbData(t, db.Path()), db.Path())
+
+	m := NewMain()
+	err := m.Run("check", "-page-Id", "0", db.Path())
```

Comment:
The parameter "0" doesn't make sense, as the default value is 0, it has the same behaviour as no parameter being provided at all. We should populate some sample data, and find out a branch page ID and at least a leaf page ID, and run `bbolt check` on the page IDs.